packages/config/src/layer2s/polygonzkevm.ts

```diff
@@ -408,7 +408,7 @@ export const polygonzkevm: Layer2 = {
         references: [
           {
             text: 'PolygonZkEvmBridge.sol - Etherscan source code, claimAsset function',
-            href: 'https://etherscan.io/address/0xb1585916487AcEdD99952086f2950763D253b923',
+            href: 'https://etherscan.io/address/0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe',
```

Comment:
```suggestion
            href: 'https://etherscan.io/address/0x0feb850b183c57534b56b7d56520133c8f9bdb65',
```

Reply:
huh, bridging/claiming is done through the proxy (0x2a3DD3EB832aF982ec71669E178424b10Dca2EDe), not the implementation (0x0feb850b183c57534b56b7d56520133c8f9bdb65), so why point to the impl?

Reply:
because we want to show the line of code that implements the functionality